impl/src/test/java/org/ehcache/GettingStarted.java

```diff
@@ -340,6 +341,24 @@ public void writeBehindCache() throws ClassNotFoundException {
     // end::writeBehindCache[]  
   }
 
+  @Test
+  public void testUpdateResourcesInRuntime() {
```

Comment:
The point of a GettingStarted example is that it is documented. Please create the necessary call outs in documentation to indicate what you are illustrating.